storage/gcp/gcp.go

```diff
@@ -770,11 +770,11 @@ func (s *spannerSequencer) assignEntries(ctx context.Context, entries []*tessera
 		// First we need to grab the next available sequence number from the SeqCoord table.
 		row, err := txn.ReadRowWithOptions(ctx, "SeqCoord", spanner.Key{0}, []string{"id", "next"}, &spanner.ReadOptions{LockHint: spannerpb.ReadRequest_LOCK_HINT_EXCLUSIVE})
 		if err != nil {
-			return fmt.Errorf("failed to read SeqCoord: %v", err)
+			return fmt.Errorf("failed to read SeqCoord: %w", err)
```

Comment:
I _suspect_ this is the only location where the `%w` will make any difference - I think all the others are going to be local ops and so can't really `Abort`.

Reply:
do you want this changed or is this just a comment? :)